sag_12r.par

```diff
@@ -1,33 +1,33 @@
-! Sagittal 12R notation. Dave Keenan & George Secor. 2006-Mar-22, updated 2012-Sep-05
-! 
+! Sagittal 12R notation. Dave Keenan & George Secor. 2006-Mar-22, updated 2020-Apr-08
+!
 max_abs_slope	6.0
 c_exp_range	-6	10
 nc_exp_range	-1	5
 fifth_size	700.0
 oct_size	2/1
 !
 !Lower bound	Symbol	Value		Short	Most common semitone fraction value
--57.14285715	(!/	8192/8505	w	-5/8
+-57.14285715	(!/	8192/8505	d	-5/8
 -51.99999999	\!/	32/33		v	-1/2
--48.00000001	\!)	35/36		u	-4/9
--42.85714285	\!~	45/46		�	-2/5
```

Comment:
Some capture zones that used to have short ASCII representations no longer have them, such as this, � being removed from \!~ without a replacement. At least, that appears to be the case from the Sagispeak thread on the forum, and our work on the Sagittal-SMuFL spreadsheet.

Was the intent for these symbols to now no longer have short ASCII? Or was there an assumption that unless otherwise specified, they should carry over? As you can see, I have assumed the former for now.

Reply:
You have assumed correctly. Cores that are neither Athenian nor single-flag, do not have short ASCII. Not sure how we want Scala to deal with that, but we, or Manuel, will come up with something.

Reply:
I have taken a note to surface this to him explicitly when we submit the changes.

By the way, I have realized that the file was in the wrong encoding before. Once I changed the encoding to `windows-1252` these �-lookin' symbols all came into their own as ð's and ¬'s and such. Not that this will be relevant anymore, with the new short ASCII.

Reply:
Thanks.